### DISCUSSION TITLE
mediamtx installed in raspi RTSP not working

### DISCUSSION THREAD

**Discussion by sravich1** [bluenviron/mediamtx#2936] [Questions]
### Question

Hi,
I tried downloading the mediamtx ..but rtsp is not working
I ran the make file and created mediamtx exe ..changed the .yml file

 paths:
568   cam:
569     source: rpiCamera
570     rpiCameraWidth: 1920
571     rpiCameraHeight: 1080
 

cam] [RPI Camera source] ready: 1 track (H264)
2024/01/24 15:25:12 INF [RTSP] [conn 10.35.62.103:57673] opened
2024/01/24 15:25:12 INF [RTSP] [conn 10.35.62.103:57673] closed: no one is publishing to path 'h264'
2024/01/24 15:25:12 INF [RTSP] [conn 10.35.62.103:57674] opened
2024/01/24 15:25:12 INF [RTSP] [session 97294194] created by 10.35.62.103:57674
2024/01/24 15:25:12 INF [RTSP] [conn 10.35.62.103:57674] closed: path of a SETUP

it says not publishing in h264....Let me know where the error is and what needs to be changed.
--Ravi

**Comment by aler9** [comment]
from the configuration and log you posted it seems that you are publishing the camera to path `/cam`, while you are trying to read from path `/h264`. Read from path `/cam` instead.

[a comment by an automated account is not shown]
